components/layout_2020/dom_traversal.rs

```diff
@@ -141,6 +146,8 @@ pub(super) enum NonReplacedContents {
     /// Content of a `::before` or `::after` pseudo-element that is being generated.
     /// <https://drafts.csswg.org/css2/generate.html#content>
     OfPseudoElement(Vec<PseudoElementContentItem>),
+    /// Workaround for input element until we properly implement display-inside
+    TextControl,
```

Comment:
```suggestion
    OfTextControl,
```
For naming consistency